packages/api-derive/test/e2e/promise.spec.ts

```diff
@@ -34,7 +35,7 @@ describe.skip('derive e2e', () => {
   });
 
   it('subscribes to newHead, retrieving the actual validator', (done) => {
-    return api.derive.chain.subscribeNewHead(({ author }) => {
+    return api.derive.chain.subscribeNewHead(({ author }: HeaderExtended) => {
```

Comment:
Is tslint throwing on these too? This means the derive typings didn't work

Reply:
The `api.derive.***` in general or this specific one? `api.derive.***` is a 100% yes, for `api.derive.chain.subscribeNewHead(({ author })` I'm only 90% sure, I'd have to roll back and double check.